docs/conf.py

```diff
@@ -14,6 +14,22 @@
 import sys
 sys.path.insert(0, os.path.abspath('../swig'))
 
+# -- Doxygen build -----------------------------------------------------------
+import subprocess
+import os
+import sys
+original_workdir = os.getcwd()
+try:
+    docs_dir = os.path.dirname(__file__)
+    os.chdir(docs_dir)
+    if not os.path.exists('doxygen/doxy'):
```

Comment:
why do we need this check?

Reply:
I suppose you already merged, but I'll comment anyway; it's there because conf.py is called twice by RTD, and Doxygen has no incremental build support. So it saves some build time.